### ISSUE TITLE
Links are not clickable

### ISSUE THREAD

**Issue by StevenWatremez** [crisp-im/crisp-sdk-android#117]
When you create a markdown link or a simple link, on Android the link is not clickable in the message. 

![Screenshot_20220523-160510](https://user-images.githubusercontent.com/11426754/170236191-cf63dbfc-0a8d-41d4-9aed-b312e178f075.jpg)
![Screenshot_20220523-160559](https://user-images.githubusercontent.com/11426754/170236199-744f3999-2372-4f6e-a5c8-709e9af3e7b9.jpg)

For exemple, the message here come from crisp automation bot when the user send a message for the first time. 

```
SDK : 1.0.9
OS : Android 11
Phone: One Plus
```

**Comment by Doc1faux** [comment]
Hello @StevenWatremez and thank you for your feedback.

Markdown is not yet supported on Android SDK but is planned to.
Currently, the priority is given to the brand new app :)

However, supporting clickable link is quite easy so I've just added it!
It will be released in the next `1.0.12`.
I let you know when it will be available.

For the Markdown support, I've created the new #118 issue to track it.
